### DISCUSSION TITLE
Should this crate be moved to `rust3ds`?

### DISCUSSION THREAD

**Discussion by ian-h-chamberlain** [rust3ds/citro3d-rs#5] [General]
@AzureMarker @Meziu since we finally have the possibility to rename stuff in the @rust3ds org, does it make sense to move this repo there as well? Obviously it's still in early stages but it depends pretty heavily on `ctru-rs` and development will happen in tandem with that crate.

Assuming we do move it, I would suggest archiving https://github.com/rust3ds/citro3d-sys and moving this with its current name (or maybe just the name `citro3d` but `-rs` might be more googleable since `citro3d` is already a thing that exists).

Thoughts?

**Comment by Meziu** [comment]
I believe `citro3d-rs` is the best option, kept with the same standard as `ctru`. You can move it already if you want to, there should be no issue.

**Reply by AzureMarker** [reply]
+1

**Reply by ian-h-chamberlain** [reply]
Ok, cool. Because the repo is a fork, I have to detach it before I can transfer back to the organization, so I've just opened a ticket with support to do that.

Once it's done, I'll transfer back and then I think the original `citro3d-sys` repo can be archived.

**Reply by ian-h-chamberlain** [reply]
Awesome, it's done! @Meziu do you want to archive the `-sys` crate now?

**Comment by Meziu** [comment]
`citro3d-sys` has been archived
